python/Ganga/Core/GangaRepository/GangaRepository.py

```diff
@@ -205,7 +205,7 @@ def _make_empty_object_(self, this_id, category, classname):
         #obj.__init__()
         obj.setNodeData({})
         obj.setNodeAttribute('id', this_id)
-
+        obj._setFlushed()
```

Comment:
I never tracked it down but I'm being explicit in-case we chose that getting an object from the Plugins performs an operation in the class ```__init__``` that can mark an object dirty. aka, setting the default backend in ```__init__``` for a Job will mark an empty job object in memory dirty and appear asif it's safe to write to disk (potentially killing whatever is on disk)